### ISSUE TITLE
[Bug]  grid.top has no effect on sunburst chart

### ISSUE THREAD

**Issue by AnotherCodeArtist** [apache/echarts#19288]
### Version

5.4.3

### Link to Minimal Reproduction

https://codesandbox.io/s/cranky-leakey-5p3h5y?file=/src/index.js

### Steps to Reproduce

1. Define a sunburst chart
2. Define a a title with text and subtext in the options object
3. Set grid.top to any value

### Current Behavior

The `grid.top` setting has no effect, which leads to an overlap of the title and the chart. In contrast to sunburst this works for bar and line charts and also sankey has an approach (a `top` property right in the `series` objects) to define a wider gap to the top.

### Expected Behavior

When setting `grid.top` this should lead to a broader gap between the chart and the top of the container.

### Environment

```markdown
- OS: 14 Sonoma
- Browser: Safari / Firefox / Chrome
- Framework: ReactJS, echarts-for-react, echarts
```

**Comment by plainheart** [comment]
Grid is only for the rectangular coordinate. In the sunburst series, you can use [series.center](https://echarts.apache.org/option.html#series-sunburst.center) option to adjust the position, for example, `center: ['50%', 200]`.

**Comment by AnotherCodeArtist** [comment]
Yeah, but this serves the purpose only to a certain extend. `series.center` moves the sunburst chart  but does not increase the container, thus, the lower part of the chart can easily become invisible, since it gets clipped by the border of the container.
I need a solution that increases the gap between title and the actual chart. This works nice for those charts supporting `grid` but also for *sankey*, which has its own `series.top` property. What about introducing such a `top` property for *sunburst* as well? Or is there a combination of properties (something like `height` and `center` that might do the trick?
Thanks for your pretty fast support!!

**Comment by plainheart** [comment]
Yeah. I think it makes sense to support these position options for `sankey` series. Added it to the backlog.

**Comment by Ovilia** [comment]
@AnotherCodeArtist Can you explain what `moves the sunburst chart but does not increase the container` means? It seems to me that `center` and `radius` is enough for the situation. Please change [the demo](https://echarts.apache.org/examples/en/editor.html?c=sunburst-simple&code=G4QwTgBAJiAuIQLwQNoCgIQN4cxAdiALYCmAXBAOQDiYI-UADiJQDS6YDGAFgJYA2UMCXwV0ePDgkTCpCpQCq-TvxIQAMiQD2bDtND8AruQgBGAKztpeHgKEixe61OvTZJygGEthgM698CAApEE4Aa11XfRAjEwAmJ2kAXyso7ETrd3lvPwCIAFlwAE9ItMwDYwpLDOlbQWFRVBrnZtcsqhDw3y18UrKJCvjWiSThiABdVpTWlzL2rx9_QIAhET60wYoAFimayetplqj5gDE4bhIwdYGYyrMABlTrOvtG8TTZtPn8kmvXTYg5imT1cn2OxA8yzAWlgF0gAAUSLBLn9rADTLsovtknpsYd0jIIfIAHL0TglEEvBqOI5tIlUJQqNTE3hhX4giRUhxNMpgulyKg5JYQVawWAUsbo4EzMbzIV5IIifAS_oQAEJMqjLGJLV4PFocYAbjQWkYsF4PSQBIgvkuvBIvgon3FjA8vkM-AARoYwL5YOsAPQBiAkIiMbggfyO62YIPSABmWk4fnkZIdsC0Vz0cfxmBg8Ao-ZAILoUF4KdQjyo5nuAFJKONKSJkWAxJQa_XWBA4vd7o29PwQJ6SPwnYlofBkfJS7wYpQ9LrRklDUA) to let me understand it better. Thanks.
